microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -145,39 +143,38 @@ After you measured and verified that the animation isn't performing well, the ne
 
    ![The Main section.](../media/evaluate-performance-performance-main.msft.png)
 
-1. There's a lot of data in the recording.  To Zoom into a single event, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **FPS**, **CPU**, and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
+1. There are a lot of data in the recording.  To zoom into a portion of the recording, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **CPU** and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
 
-   ![Zoom into an event.](../media/evaluate-performance-performance-main-zoomed.msft.png)
+   ![Zoom into a section.](../media/evaluate-performance-performance-main-zoomed.msft.png)
 
-   Another way to zoom is to put focus on the **Main** section, click the background or an event, and press `W`, `A`, `S`, or `D`.
+   Another way to change the selected area is to put focus on the **Main** section, click the background or an event, and press `W` to zoom in, `A` to zoom out, `S` to move selection left, or `D` to move selection right.
 
-1. Focus on the red triangle in the top-right of the **Animation Frame Fired** event.  Whenever a red triangle is displayed, it's a warning that there may be an issue related to the event.
+1. Select a **Animation Frame Fired** event.  When a red triangle is displayed at the top right of an event, it's a warning that there may be an issue related to the event.
```

Comment:
```suggestion
1. Select an **Animation Frame Fired** event.  When a red triangle is displayed at the top right of an event, it's a warning that there might be an issue related to the event.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may - very much an area of judgment.  'may' seems ok, and is often used in this repo, but I'd probably write 'might' because it's inherently unambiguous.